assets/app/view/log.rb

```diff
@@ -31,9 +31,7 @@ def render
         on: { scroll: scroll_handler },
         style: {
           overflow: 'auto',
-          height: '200px',
-          width: '100%',
-          padding: '0.5rem',
+          padding: '0 0.5rem',
```

Comment:
this will mess up emails

Reply:
Ok, will change. But mails should have a text version without HTML/CSS ;)